diff -rupN orig/CMakeLists.txt new/CMakeLists.txt
--- orig/CMakeLists.txt	2011-07-21 14:53:58.000000000 +0200
+++ new/CMakeLists.txt	2011-07-21 15:11:03.131921022 +0200
@@ -150,5 +150,4 @@ add_subdirectory("src")
 # Install Rules
 ################################################################################
 
-install(FILES "LICENSE.TXT" "README.TXT" DESTINATION ".")
-install(FILES "cmake/FindVL.cmake" DESTINATION "cmake")
+install(FILES "cmake/FindVL.cmake" DESTINATION "share/cmake/Modules")
diff -rupN orig/data/CMakeLists.txt new/data/CMakeLists.txt
--- orig/data/CMakeLists.txt	2011-07-21 14:53:58.000000000 +0200
+++ new/data/CMakeLists.txt	2011-07-21 15:11:35.668921034 +0200
@@ -19,6 +19,6 @@ project(DATA)
 
 option(VL_INSTALL_DATA "Install sample data" OFF)
 if(VL_INSTALL_DATA)
-	install(DIRECTORY 3rdparty font glsl images models text volume mol DESTINATION "data/")
-	install(FILES README.TXT ztest.zip DESTINATION "data/")
+	install(DIRECTORY 3rdparty font glsl images models text volume mol DESTINATION "share/vl/data/")
+	install(FILES README.TXT ztest.zip DESTINATION "share/vl/data/")
 endif()
